controller/src/main/java/com/anna/controller/StudentsController.java

```diff
@@ -34,13 +34,24 @@ public StudentsController(StudentsService studentService) {
     this.studentService = studentService;
   }
 
+  /**
+   * Get student's list.
+   *
+   * @param page number of page
+   * @param size number of students per page
+   * @param minBirthDate min value for sort
+   * @param maxBirthDate max value for sort
+   * @return list
+   */
   @JsonView(View.Student.class)
-  @GetMapping("/students")
+  @GetMapping(value = "/students")
   @ResponseStatus(HttpStatus.OK)
   public List<Student> getStudents(
+      @RequestParam(value = "page", required = false, defaultValue = "1") Integer page,
```

Comment:
add validation for parameters not to be equal or less then zero (if fails = 400 Bad request)
add case if page > then possible - 404 not found